etna/core/blocks/cta.py

```diff
@@ -0,0 +1,80 @@
+from wagtail.core import blocks
+
+# from etna.collections.blocks import (
+#    TimePeriodExplorerIndexBlock,
+#    TopicExplorerIndexBlock,
+# )
+#
+from etna.core.blocks import (  # ContentImageBlock,
+    PageListBlock,
+    SectionDepthAwareStructBlock,
+)
+
+# from wagtail.images.blocks import ImageChooserBlock
+
+
+class TimePeriodBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(max_length=100, default="Explore by time period")
+    sub_heading = blocks.CharBlock(
+        max_length=200,
+        default="Discover 1,000 years of British history through time periods including:",
+    )
+    page = blocks.PageChooserBlock(page_type="collections.TimePeriodExplorerIndexPage")
+
+    class Meta:
+        template = "collections/blocks/time_period_explorer_new.html"
```

Comment:
1. Don't know how long a filename will stay `_new` or how to differentiate it from one without `_new`; could we suffix other than `_new`?
2. After adding `Time Period` in for root page - home page, was not able to see `time_period_explorer_new.html` in Debug Tool Bar in Preview. Is there any other way to view it?